jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/StringUtil.java

```diff
@@ -259,6 +259,23 @@ public static boolean asciiEqualsIgnoreCase(String string, String other)
         return true;
     }
 
+    /**
+     * Check for string equality.
+     * @param string The string to check
+     * @param other The other string to check
+     * @return true if the strings are equal.
+     */
+    public static boolean equals(String string, String other)
+    {
+        if (string == null)
+            return other == null;
+
+        if (other == null)
+            return false;
+
+        return string.equals(other);
+    }
```

Comment:
isn't this just `Objects.equals(Object, Object)` ?

Reply:
uh ... yeah?

Reply:
Removed this new StringUtils.equals()